dashboard/app/models/script.rb

```diff
@@ -1367,7 +1368,8 @@ def summarize(include_lessons = true, user = nil, include_bonus_levels = false)
       background: background,
       is_migrated: is_migrated?,
       scriptPath: script_path(self),
-      showCalendar: show_calendar
+      showCalendar: show_calendar,
```

Comment:
is_migrated ? show_calendar : false #prevent calendar from showing for non-migrated scripts for now.